tests/test_service.py

```diff
@@ -1056,3 +1056,34 @@ def testGetUserDerivativeNonExisting(self, mockGet):
         res = self.argus.derivatives.get(derivativeID_1)
         self.assertTrue(not res)
 
+    @mock.patch('requests.Session.get', return_value=MockResponse(json.dumps(derivative_1_D), 200))
+    def testGetUserDerivatives(self, mockGet):
+        res = self.argus.derivatives.get_user_derivatives(mockGet)
+        self.assertTrue(res is not None)
+        self.assertIn((os.path.join(endpoint, "derivatives/meta"),), tuple(mockGet.call_args))
+
+    @mock.patch('requests.Session.get', return_value=MockResponse(json.dumps(derivative_1_D), 200))
+    def testGetUserDerivativesPage(self, mockGet):
+        res = self.argus.derivatives.get_user_derivatives_page(mockGet)
+        self.assertTrue(res is not None)
+        self.assertIn((os.path.join(endpoint, "derivatives/meta/user"),), tuple(mockGet.call_args))
+
+    @mock.patch('requests.Session.get', return_value=MockResponse(json.dumps(derivative_1_D), 200))
+    def testGetUserDerivativesCount(self, mockGet):
+        res = self.argus.derivatives.get_user_derivatives_count(mockGet)
+        self.assertTrue(res is not None)
+        self.assertIn((os.path.join(endpoint, "derivatives/meta/user/count"),), tuple(mockGet.call_args))
+
+    @mock.patch('requests.Session.get', return_value=MockResponse(json.dumps(derivative_1_D), 200))
+    def testGetSharedUserDerivatives(self, mockGet):
+        res = self.argus.derivatives.get_shared_user_derivatives(mockGet)
+        self.assertTrue(res is not None)
+        self.assertIn((os.path.join(endpoint, "derivatives/meta/shared"),), tuple(mockGet.call_args))
+
+    @mock.patch('requests.Session.get', return_value=MockResponse(json.dumps(derivative_1_D), 200))
+    def testGetSharedUserDerivativesCount(self, mockGet):
+        res = self.argus.derivatives.get_shared_user_derivatives_count(mockGet)
+        self.assertTrue(res is not None)
+        self.assertIn((os.path.join(endpoint, "derivatives/meta/shared/count"),), tuple(mockGet.call_args))
+
+
```

Comment:
nit: extra blank lines not needed.

```suggestion
        self.assertIn((os.path.join(endpoint, "derivatives/meta/shared/count"),), tuple(mockGet.call_args))
```

Reply:
Done @hdara-sfdc 
But the CI is still failing, how do I get that passed? Complains that "One or more contributors need to sign the CLA"